core/src/controllers/rows.js

```diff
@@ -106,7 +112,9 @@ const listTableRows = async (req, res) => {
     whereString += filters
       .map(
         (filter) =>
-          `${tableName}.${filter.field} ${filter.operator} '${filter.value}'`
+          `${tableName}.${filter.field} ${filter.operator} ${
+            filter.value != null ? filter.value : ''
```

Comment:
We can use `||` here, `filter.value || ''`.

Reply:
@thevahidal My first approach was not correct, but i have tested the above approach and it would also fail, the thing is we can't put the `''` value if `filter.value` is `null` that will result with incorrect SQL query, the correct code would be something like this 

```js
.map((filter) =>
        filter.value !== null
          ? `${tableName}.${filter.field} ${filter.operator} '${filter.value}'`
          : `${tableName}.${filter.field} ${filter.operator}`
      )

```